src/main/java/nl/wearefrank/openapifrankadapter/OpenapiFrankadapterApplication.java

```diff
@@ -53,42 +53,79 @@ public static void main(String[] args) {
 
     }
 
-    @PostMapping(value = "/receiver-file", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
-    public ResponseEntity<Resource> postFileReceiver(@RequestParam("file") MultipartFile file) throws IOException, SAXException {
-        // Check if it's a JSON file
-        if (!file.getContentType().equals("application/json") && !file.getContentType().equals("application/yaml")) {
+    public static boolean checkUrl(String url) throws IOException {
```

Comment:
`checkUrl` maakt een HTTP request om te kijken wat de content-type is. Als die goed is wordt in `downloadFileFromUrl` nog eens een request gemaakt. Dit moet 1 request worden.

Ik vind de naam `checkUrl` ook niet echt heel duidelijk. Ik snap hieruit niet wat het doet.